models/shopify__discounts.sql

```diff
@@ -0,0 +1,90 @@
+with discount as (
+
+    select *
+    from {{ var('shopify_discount_code') }}
+),
+
+price_rule as (
+
+    select *
+    from {{ var('shopify_price_rule') }}
+),
```

Comment:
I noticed there are a number of other price_rule type tables in our ERD. Do you feel these are tables we should consider for this end model? 
![image](https://user-images.githubusercontent.com/74217849/211907358-b101c9d2-0b95-4c83-b949-d9c16c968eb3.png)

Reply:
yeah i considered these but felt that they didn't add too much for their extra complexity. they only join with a subset of price rules (ones that either have `allocation_method = 'each' or customer_selection = 'prerequisite'`). 

i think the only thing we'd do with these tables is aggregate the products/collections/variants/countries/customers associated with the price rule. that could be helpful i suppose...what do you think joe?

Reply:
Yeah I saw these and had a bit of a shudder in terms of what value they would actually bring, but how much complexity they would even more so add.

Let's opt to exclude them and consider adding them if users engage and want to add them into a future version.